datasets/bbga/bbga.json

```diff
@@ -0,0 +1,204 @@
+{
+  "type": "dataset",
+  "id": "bbga",
+  "title": "BBGA",
+  "status": "beschikbaar",
+  "description": "Basisbestand Gebieden Amsterdam",
+  "version": "0.0.1",
+  "crs": "EPSG:4326",
+  "tables": [
+    {
+      "id": "indicatorenDefinities",
+      "title": "Metedata Indicatoren en Definities",
+      "type": "table",
+      "schema": {
+        "$id": "https://github.com/Amsterdam/schemas/bbga/indicatoren_definities.json",
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "identifier": "variabele",
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "id",
```

Comment:
Omdat je in de tabel geen id kolom hebt, zou je deze m.i. kunnen vervangen door variabele. Of werkte dit bij jou wel goed?

Reply:
Goede, ga ik vervangen door `"variabele"` en kijken wat het effect is.

Reply:
Met "variabele" ipv "id" opgenomen in "required" werkt de boel nog steeds goed.